pkg/refproto/ssl_game_event.proto

```diff
@@ -178,6 +178,8 @@ message GameEvent {
         // an angle of 0 rad (  0°) means, the bots barely touched each other
         // an angle of π rad (180°) means, the bots crashed frontal into each other
         optional float crash_angle = 6;
+        // the location of the ball at the moment when this foul occurred
```

Comment:
Crashes are currently designated as non-stopping fouls, do we even need the ball position?

Reply:
you are right, that's not needed.